### PR TITLE
Permissions, HTTPS, BABAJKA_SECRET

### PR DESCRIPTION
**Pull request by UladBohdan**
* a script to grant permissions to users
* HTTPS support
* BABAJKA_SECRET deprecated and replaced with command-line argument

### CODE REVIEW

**Comment by UladBohdan** [comment]
[to add description to every permission with have (to have it documented)](https://trello.com/c/mCZ8yasw/214-to-add-description-to-every-permission-with-have-to-have-it-documented)

**Comment by UladBohdan** [comment]
[empty permissions object for EVERY user (now it's missing for some)](https://trello.com/c/9aPevVic/205-empty-permissions-object-for-every-user-now-its-missing-for-some)

File: src/constants/permissions.js [hunk 1]
```diff
@@ -1,3 +1,12 @@
+// A description of permissions we're using aroung the project:
+// * canCreateArticle - the one with this permission is treated as an Author:
+//    he/she can create new articles and edit those for which he/she is an author
+//    (article's authorId matches).
+// * canManageArticles - the one with this permission is a contentManager:
+//    he/she can edit and remove existing articles bun CAN NOT create new.
+// * canManageUsers - the one with this permission can create, edit and remove
+//    and view the full list of users existing.
```
Comment (Drapegnik) [inline src/constants/permissions.js]: TODO: use some standart for building docs from comment in the future, for example [`jsdoc`](https://github.com/jsdoc3/jsdoc)

Comment (Drapegnik) [reply src/constants/permissions.js]: https://trello.com/c/dtNGlzjS/221-add-jsdoc-or-other-tool-for-docs-generation

File: src/db/scripts/grantPermissions.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
+/* eslint-disable no-console */
+
+// Should be called with two options: email, role.
+// Role is one from constants/permissions.
```
Comment (Drapegnik) [inline src/db/scripts/grantPermissions.js]: @UladBohdan please add here example of call or, better, `console.log` for calling without / with bad params

**Pull request merged**